soperator/mlperf/gpt3-impl-4.0-nvidia/Dockerfile

```diff
@@ -121,6 +121,21 @@ RUN rm -rf /opt/hpcx/nccl_rdma_sharp_plugin && \
 RUN pip install huggingface_hub==0.23.2
 RUN pip install -v "transformers<=4.40.2"
 
+## Reinstall NCCL to the latest version
```

Comment:
Why that commented?

Reply:
Because it's not used by default. Just saved it because I may need it in the future